sos/plugins/openstack_aodh.py

```diff
@@ -34,15 +35,28 @@ class OpenStackAodh(Plugin, RedHatPlugin):
     requires_root = False
 
     def setup(self):
-        self.add_copy_spec("/etc/aodh/")
+        self.add_copy_spec([
+            "/etc/aodh/",
+            "/var/lib/config-data/puppet-generated/aodh/etc/aodh/",
+            "/var/lib/config-data/puppet-generated/aodh/etc/httpd/conf/",
+            "/var/lib/config-data/puppet-generated/aodh/etc/httpd/conf.d/",
+            "/var/lib/config-data/puppet-generated/aodh/etc/httpd/"
+            "conf.modules.d/*.conf",
+            "/var/lib/config-data/puppet-generated/aodh/etc/my.cnf.d/"
```

Comment:
Given that these all have a common prefix, why not store the `/var/lib/config-data/puppet-generated` path in a local, e.g. `var_puppet_generated`, and then cons the rest of the string on in the list?

This would avoid the need to break paths over multiple lines which reduces the readability of the list and makes it harder to parse visually.